website/source/docs/providers/aws/d/redshift_account_id.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "aws"
+page_title: "AWS: aws_redshift_account_id"
+sidebar_current: "docs-aws-datasource-redshift-account-id"
+description: |-
+  Get AWS Redshift Account ID
+---
+
+# aws\_redshift\_account\_id
+
+Use this data source to get the Account ID of the [AWS Redshift Account](http://docs.aws.amazon.com/redshift/latest/mgmt/db-auditing.html#db-auditing-enable-logging)
+in a given region for the purpose of negotiating with other AWS services.
```

Comment:
how about `for the purpose of allowing Redshift to store audit logging data in S3` or something more concise... just to differentiate between the CloudTrail ones and these.